client/state/post-types/taxonomies/reducer.js

```diff
@@ -61,18 +59,10 @@ export function items( state = {}, action ) {
 				},
 			} );
 
-		case DESERIALIZE:
-			if ( isValidStateWithSchema( state, itemsSchema ) ) {
-				return state;
-			}
-
-			return {};
-		case SERIALIZE:
+		default:
 			return state;
 	}
-
-	return state;
-}
+} );
```

Comment:
Note that we also have an option of calling
```jsx
items.schema = itemsSchema;
```
But what's here is fine too 👍

Reply:
but then it wouldn't pass tests. I think wrapping reducers with `withSchemaValidation` could be a better idea going forward

Reply:
I mention this in the PR description -- `items.schema` would require an update of tests, so I went the easy route. Thanks for reviewing!

Reply:
Anyway, my goal here is to move the schema validation from the individual reducers to the framework, so that I can eventually optimize it there at one place.

Both `withSchemaValidation` and `items.schema` achieve that goal equally well.